redis/redis.go

```diff
@@ -368,6 +387,10 @@ func (store *redisFeatureStoreCore) InitInternal(allData map[ld.VersionedDataKin
 	c := store.getConn()
 	defer c.Close() // nolint:errcheck
 
+	if store.auth != "" {
+		_ = c.Send("AUTH", store.auth)
+	}
+
```

Comment:
I'm not sure this is the right place for this.

AUTH needs to be the first command that we send - but is that per connection, or per connection pool? If it's per connection then we would need to do it every time we called `store.getConn`, i.e. for every kind of operation. But then we'd have no way of knowing if this is a new connection or a reused one.

I think possibly the way we're supposed to do this is by specifying the DialPassword option when we call `redis.DialURL()`.

Reply:
You're right, this should've been per Connection.

I just updated the commit to use the `redis.DialPassword` option in `redis.DialURL()`.